.github/dependabot.yml

```diff
@@ -1,3 +1,8 @@
+# The dependabot checks are set on a schedule to run weekly and daily.
+# The time of day is set to be early in the morning to avoid working hours.
+# Furthermore, the time is set arbitrarily to avoid the general "high tide"
+# of all GH Actions being run at the top of the hour.
+
```

Comment:
Suggested rephrase: Dependabot checks run on both a weekly and daily schedule. They're set to run early in the morning, outside of typical working hours. This timing is also deliberately offset from the start of the hour to prevent overlap with peak GitHub Actions activity [honestly yours is already plenty clear though]

Reply:
This seems like what I've already wrote. Will keep what I have.